helm/libero-reviewer/values.yaml

```diff
@@ -28,3 +30,12 @@ postgresql:
     enabled: false
   postgresqlDatabase: "postgres"
   postgresqlPassword: "change_this_password_or_else"
+
+# todo, set this to proper values
+continuumLogin:
```

Comment:
Is there a ticket raised to do this?

Reply:
not yet but we should do a proper review anyway

/cc @giorgiosironi